pkgs/tools/networking/mosh/default.nix

```diff
@@ -25,6 +25,12 @@ stdenv.mkDerivation rec {
       url = "https://github.com/mobile-shell/mosh/commit/e5f8a826ef9ff5da4cfce3bb8151f9526ec19db0.patch";
       sha256 = "15518rb0r5w1zn4s6981bf1sz6ins6gpn2saizfzhmr13hw4gmhm";
     })
+    # Fix build with bash-completion 2.10
+    # https://github.com/mobile-shell/mosh/pull/1095
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/mobile-shell/mosh/pull/1095.patch";
```

Comment:
This is pointing to a mutable location - like `https://github.com/mobile-shell/mosh/pull/1095/commits/$commitID.patch`

But I also see https://github.com/mobile-shell/mosh/pull/1095 is still a draft an WIP.

Reply:
This url should be safe to use: https://github.com/mobile-shell/mosh/commit/ebad195db0e8763ebb7ef70aad906c826644abf6.patch
It will also work if the PR is changed.